pkg/cmd/shared/handler/indices/config_mock.json

```diff
@@ -0,0 +1,134 @@
+{
```

Comment:
@clemfromspace 
From what I understood, this will never be in the final build. However, I'm not sure this is the best practice to test that feature: wdyt?

Reply:
IMHO, it will be cleaner if you put this file under a sub-directoy like `test_artifacts` or something similar.